### PR TITLE
net: lib: nrf_cloud: update nrf cloud host name

### PR DESCRIPTION
**Pull request by jayteemo**
The value of Kconfig option  NRF_CLOUD_HOST_NAME is now
simply "mqtt.nrfcloud.com"
IRIS-4190

Signed-off-by: Justin Morton <justin.morton@nordicsemi.no>

### CODE REVIEW

**Review by glarsennordic — APPROVED** [review]
LGTM

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -465,6 +465,7 @@ Libraries for networking
         The data is now sent to the application even if no ``"config"`` section is present.
       * The application can now send shadow updates earlier in the connection process.
       * nRF Cloud error message responses to location service MQTT requests are now handled.
+      * The value of :kconfig:option:`NRF_CLOUD_HOST_NAME` option is now simply "mqtt.nrfcloud.com".
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:468]:
```suggestion
      * The value of the :kconfig:option:`NRF_CLOUD_HOST_NAME` option is now "mqtt.nrfcloud.com".
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

**Comment by martiles** [comment]
The old NRF_CLOUD_HOST_NAME will still be valid to use for the foreseeable future right?

**Comment by jayteemo** [comment]
> The old NRF_CLOUD_HOST_NAME will still be valid to use for the foreseeable future right?

yes, it will remain valid.

**Review by plskeggs — APPROVED** [review]

**Pull request merged**